### PR TITLE
Add kube-rbac-proxy package

### PR DESCRIPTION
**Pull request by sandinv**
#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [X] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

File: kube-rbac-proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: kube-rbac-proxy
+  version: 0.15.0
+  target-architecture:
+    - x86_64
+    - aarch64
```
Comment (ajayk) [inline kube-rbac-proxy.yaml:6]: This is not required , by default it builds for both x86 and aarch64 
```suggestion
```

Comment (ajayk) [reply kube-rbac-proxy.yaml:6]: on wolfi currently we build only for x86_64 and aarch64

**Review by ajayk — APPROVED** [review]

**Comment by ajayk** [comment]
Thanks @sandinv for your contribution

**Comment by sandinv** [comment]
thank you so much for your help and support @ajayk

**Pull request merged**